exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/jupysql_init.ipynb

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">model loading functions
</pre><p>I think, this a copy paste typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yep, well spotted! Thanks</p>